test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

```diff
@@ -20,6 +20,7 @@ describe('AuthorisationCaseEvent', () => {
         'AuthorisationCaseEvent-general-email-nonprod',
         'AuthorisationCaseEvent-general-referral-nonprod',
         'AuthorisationCaseEvent-alternative-service-nonprod',
+        'AuthorisationCaseEvent-alt-service-process-server-nonprod',
```

Comment:
Aren't 'AuthorisationCaseEvent-alternative-service-nonprod' and 'AuthorisationCaseEvent-alt-service-process-server-nonprod' supposed to be the same feature flag?
I think we're going live with these together and it would be simpler if they're the same.

Reply:
My understanding was that, as these are two different epics, they should be different

Reply:
Might be safer to have them separated if we're not sure, unless Jeremy can confirm I guess.